dev-commands.sh

```
@@ -0,0 +1,297 @@
+# Clean Android build
+rebuild-android() {
+  echo "Cleaning Android build..."
+  cd android && ./gradlew clean
+  echo "✅ Android cleaned"
+  cd ..
+}
+
+# Clean iOS build
+rebuild-ios() {
+  echo "Cleaning iOS build..."
+  cd ios
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)